File: julia/fizzbuzz.jl
for i in 1:100
    if i % 3 == 0 && i % 5 == 0
      print("fizzbuzz\n")
    elseif i % 3 == 0
      print("fizz\n")
    elseif i % 5 == 0
      print("buzz\n")
    else
      print("", i)
      print("\n")
    end
  end

File: pascal/fizzbuzz.pas
program fizzBuzz;
var
    i: integer;

begin
    for i := 1 to 100 do

        begin
            if (i mod 3=0) and (i mod 5 = 0) then
                writeln('fizzbuzz')
            else if (i mod 3 = 0) then
                writeln('fizz')
            else if (i mod 5 = 0) then
                writeln('buzz')
            else
                writeln(i);
        end;
end.
